core/tests/test_utils.py

```diff
@@ -3609,151 +3590,6 @@ def save_new_story(
         story_services.save_new_story(owner_id, story)
         return story
 
-    def save_new_story_with_story_contents_schema_v1(
```

Comment:
Were we just not using all these functions?

Reply:
correct